add researchers page for jupyter

eatyourpeas opened the pull request:
### Overview

Adds documentation and links to jupyter for RCPCHGrowth

pacharanero commented:
Nice work, just a few comments:

1. "The RCPCHGrowth algorithm builds on [LMSGrowth](https://www.healthforallchildren.com/t/191) excel plug in developed by Prof Tim Cole and Huiqi Pan." to me this reads as though we have **literally** built on the Excel plugin, as in, our RCPCHGrowth package is also an Excel plugin or depends on the LMSGrowth package, neither of which are true.
2. The Binder link works but the Binder package doesn't work. I know this is WIP but maybe we should develop these pages behind the scenes with Markdown in the docs site without 'enabling' them in `mkdocs.yml` - and unveil them when they are ready?
3. I wonder if we should start allowing a few **working** days (M-F) between raising PRs and merging them, to allow other team to offer feedback, tweak etc. I do see these PR notifications in email but my working week is primarily M-F so I don't always have the time to comment, review etc.

eatyourpeas commented:
Of course - sorry. Although this is in the live main branch it is not released as yet. The main changes in this PR were:
1. deprecate `setup.py` in favour of `pyproject.toml`. This has been on the list for a while and has been the recommended way to build packages since 2020 (see [Simon Wallinson blog](https://til.simonwillison.net/python/pyproject)]
2. Add python notebook support. 

It was meant to be a start that we would develop as time went by. They worked fine in my docker so I updated the docs at that point but realized later that the Binder link did not work. Hence then the question about having a separate repository. I updated the docs before I knew that.

**So...**
- I will pause here for the moment as you suggest. 
- I will tidy the docs to be more meaningful and correct and remove the binder link for the moment. 
- I will also remove the binder link from the readme here. 
- I have bumped the minor version of the package - once you/others are happy I will create a new release. Even once released, it will not hit the server until the package version there is updated and I wanted to test that locally first.
